packaging/vm-deployment/bpfd.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=bpfd - loading daemon
+After=network.target
+
+[Service]
+ExecStart=/usr/sbin/bpfd
+MemoryAccounting=true
+MemoryLow=infinity
+MemoryMax=infinity
+User=vagrant
```

Comment:
Should be `bpfd` user and group here. Need to have ansible create them if they don't exist.

Reply:
Done!